tarpc/src/macros.rs

```diff
@@ -569,6 +581,20 @@ mod functional_test {
         assert_eq!(3, client2.add(1, 2).get().unwrap());
     }
 
+    #[test]
+    fn async_try_clone_unix() {
+        let handle = Server.spawn_with_config(UnixTransport("/tmp/test"),
```

Comment:
Hm, I must have been looking at a different crate last time. Tempdir should work.